cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -1015,7 +1015,9 @@ void handle_fli_callback(void *data)
     FliProcessCbHdl *cb_hdl = (FliProcessCbHdl*)data;
 
     if (!cb_hdl) {
-        LOG_CRITICAL("FLI: Callback data corrupted: ABORTING");
+        LOG_ERROR("FLI: Callback data corrupted: ABORTING");
+        gpi_embed_end();
```

Comment:
Since this one really is a program termination, I think `critical` would be very appropriate here.